modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/NotificationsSettingsFragment.kt

```diff
@@ -445,6 +402,48 @@ class NotificationsSettingsFragment :
         }
     }
 
+    private fun setupEnabledNotifications() {
+        launch(Dispatchers.Default) {
+            val enabled = settings.notifyRefreshPodcast.flow.value
```

Comment:
Do you think it's worth having a `value` method that just calls `flow.value` as we seem to call it lots? Feel free to ignore this.

Reply:
My initial thought was to intentionally not add something like that because I wanted to provide a push toward reacting to the flow instead of just fetching the latest value. There are enough places where we just get the latest value though that you're probably right that it makes sense to just make that more convenient. I'll make this change in a separate PR. Thanks for bringing this up.